app/views/layouts/application.html.erb

```diff
@@ -30,14 +32,11 @@
 
   <header class="govuk-header " role="banner" data-module="header">
     <div class="govuk-header__container govuk-width-container">
-
```

Comment:
Removing these is good but it might be best doing it in a separate PR, so that this PR is only about the alpha banner :) Just leave it in for now though if you want 👍